packages/wonder-blocks-toolbar/docs.md

```diff
@@ -0,0 +1,192 @@
+> TODO(scottgrant): Finish building, and write more thorough docs.
+
+## Content toolbar
+
+### Standard view
+
+```js
+<Toolbar
+	contentKind="Video"
+	showClose={true}
+	title="Counting with small numbers"
+/>
+```
+
+### Content completed view
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+
+const startExercise = <Button>Next Video</Button>;
+
+<Toolbar
+	contentKind="Video"
+	rightContent={[startExercise]}
+	showClose={false}
+	title="Counting with small numbers"
+/>
+```
+
+## Exercise toolbar
+
+### Generic header
+
+```js
+const Link = require("@khanacademy/wonder-blocks-link").default;
+
+// TODO(scottgrant): This is a placeholder icon. The icon in Zeplin was rotated.
```

Comment:
FYI `Link` is eventually supposed to have a `caret` prop so this should be really easy in the future.

Reply:
Whew. That's great to hear!